event-land/scss/_variables.scss

```diff
@@ -0,0 +1,17 @@
+/*------------------------------------*\
+  #COLORS
+\*------------------------------------*/
+
+$primary-color: #50cb8b;
+$primary-color-lighter: #effff7;
+$secondary-color: #ffffff;
+$secondary-color-darker: #f1f1f1;
+$tertiary-color: #24222c;
```

Comment:
I recommend to use camelCase name convention, and please update the code convention page as well.

Reply:
Done